charts/etos/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+Put some notes here on how to verify an ETOS deployment
```

Comment:
Indeed. Future issue? If so, write the issue now :)

Reply:
Unsure what we should suggest here. Perhaps just a simlple text askinf to check API end points. Future work could involve instructing to run an actual sanity check test suite e.g dogfooding. I'm open to suggestions here.

Reply:
I like the idea that we can execute ETOS tests in ETOS as a sanity check, if possible.
Write an issue for it instead

Reply:
That would require us to add some default providers built-in to ETOS for it to work.
An IUT Provider that just provides a number of IUTs (configured via dataset from etos-client).
An Execution Space Provider that provides space in kubernetes.
A Log Area that stores in redis or something?

We would also need to send a few ArtifactCreated, ArtifactPublished events during deployment for the tests to work, I think.

Reply:
My idea was to for the note to provide instructions on how to run the tests on their newly deployed cluster, as the notes are shown upon successful deployment. But we might just begin with asking the installer/dfeployer to check endpints of etos-api and env-provider.

Reply:
My idea as well. But my idea revolves on actually executing ETOS.
Let's start with your idea. It's easier.

Endpoints:
HEAD etos-api/ (verify {"Connection": "SUCCESSFUL"})
GET environment-provider/ (verify bad request)  - Should be fixed in the future.
kubectl logs po/suite-starter (Verify logs are quiet (means rabbitmq connection succeeded))  - Should be fixed in the future.

Reply:
We should probably implement some sort of health check system for ETOS so that it's easier to validate.
Could possibly be executed as part of deployment?

Reply:
I'll put something in the notes with the end-points and I'll file an issue regarding health check upon deployment.

Reply:
Fixed.